modules/sdk-coin-avaxc/src/avaxc.ts

```diff
@@ -837,7 +854,9 @@ export class AvaxC extends BaseCoin {
           );
         }
       }
-      // TODO(BG-59774): Implement verifySignature without using Private Key
+      if (!(await this.verifySignatureForAtomicTransaction(tx))) {
```

Comment:
can't we just verify it in the sdk? 
i/e/ move the logic to sdk

Reply:
Verification is being implemented here, right?
This is SDK. 
I don't get you.